<?xml version="1.0"?>
<launch>
    <arg name="robot_name" default="rb1_base"/>
    <arg name="neighbours_topic" default="/neighbourhood"/>
    <arg name="fixed_frame" default="map"/>
    <arg name="base_frame" default="rb1_base_a_base_footprint"/>
    <arg name="prohibition_layer_srv" default="/rb1_base_a/move_base/global_costmap/costmap_prohibition_layer/update_zones"/>
    <arg name="desired_freq" default="50"/>

    <node name="neighbour_positioner" pkg="neighbour_positioner" type="neighbour_positioner_node" output="screen">
        <param name="robot_name" value="$(arg robot_name)"/>
        <param name="neighbours_topic" value="$(arg neighbours_topic)"/>
        <param name="fixed_frame" value="$(arg fixed_frame)"/>
        <param name="base_frame" value="$(arg base_frame)"/>
        <param name="prohibition_layer_srv" value="$(arg prohibition_layer_srv)"/>
        <param name="desired_freq" value="$(arg desired_freq)"/>
    </node>

</launch>
